geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/P2PMessagingConcurrencyDUnitTest.java

```diff
@@ -126,24 +194,41 @@ private void configure(
        * all combinations of flags with buffer sizes:
        * (equal), larger/smaller, smaller/larger, minimal
        */
-      "true, true, 32768, 32768",
-      "true, true, 65536, 32768",
-      "true, true, 32768, 65536",
-      "true, true, 1024, 1024",
-      "true, false, 32768, 32768",
-      "true, false, 65536, 32768",
-      "true, false, 32768, 65536",
-      "true, false, 1024, 1024",
-      "false, true, 32768, 32768",
-      "false, true, 65536, 32768",
-      "false, true, 32768, 65536",
-      "false, true, 1024, 1024",
-      "false, false, 32768, 32768",
-      "false, false, 65536, 32768",
-      "false, false, 32768, 65536",
-      "false, false, 1024, 1024",
+      "true, true, true, 32768, 32768",
+      "true, true, true, 65536, 32768",
+      "true, true, true, 32768, 65536",
+      "true, true, true, 1024, 1024",
+      "true, true, false, 32768, 32768",
+      "true, true, false, 65536, 32768",
+      "true, true, false, 32768, 65536",
+      "true, true, false, 1024, 1024",
+      "true, false, true, 32768, 32768",
+      "true, false, true, 65536, 32768",
+      "true, false, true, 32768, 65536",
+      "true, false, true, 1024, 1024",
+      "true, false, false, 32768, 32768",
+      "true, false, false, 65536, 32768",
+      "true, false, false, 32768, 65536",
+      "true, false, false, 1024, 1024",
+      "false, true, true, 32768, 32768",
+      "false, true, true, 65536, 32768",
+      "false, true, true, 32768, 65536",
+      "false, true, true, 1024, 1024",
+      "false, true, false, 32768, 32768",
+      "false, true, false, 65536, 32768",
+      "false, true, false, 32768, 65536",
+      "false, true, false, 1024, 1024",
+      "false, false, true, 32768, 32768",
+      "false, false, true, 65536, 32768",
+      "false, false, true, 32768, 65536",
+      "false, false, true, 1024, 1024",
+      "false, false, false, 32768, 32768",
+      "false, false, false, 65536, 32768",
+      "false, false, false, 32768, 65536",
+      "false, false, false, 1024, 1024",
```

Comment:
the set of configurations doubled when I introduced the ordered/unordered delivery switch

Reply:
reverted